### PR TITLE
#U22 feat: adds blog section for mobile

### PR DESCRIPTION
**Pull request by chrisostome01**
# PR Description

This pull request aims to add blogs section for mobile devices

# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you  Make sure you check every completed task. list them in this style

- [x]  Individual blogs have a shadow
- [x]  fixed width and height on the blog
- [x]  the justification of contents in the consistent events regardless of the height of the text
- [x]  Only show one blog and 15% of the other blog
- [x]  the remaining blogs are viewed when we scroll
- [x]  the blogs are overflowing initially only on the right side but on both sides when we scroll horizontally

# Description of tasks that were expected to be completed and that are not yet completed

- [x] none.....

# How this maybe tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependences 
2. run `npm run dev`
3. navigate to localhost:3000 

# Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case its appropriate and applicable

# Please check this Checklist before you submit your PR:

- [x ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -15,3 +15,15 @@
...
+/* hide the scoll but keep the scroll functioning */
+.hide-scroll {
+  overflow-y: scroll;
+  scrollbar-width: none; 
+  -ms-overflow-style: none;
+}
+.hide-scroll::-webkit-scrollbar { 
+  width: 0;
+  height: 0;
+}
```
Comment (Yvad60) [inline styles/globals.css]: nice one here !

File: components/Sections/BlogPost.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import Blog from "../ui/Blog"
+import Wrapper from "../wrapper/Wrapper"
+import blogData from "/utils/blogData"
+
+export default function BlogPost() {
+    const [blogs , setBlogs] = useState([])
+
+    useEffect(() => {
+        setBlogs(blogData)
+    },[])
```
Comment (Yvad60) [inline components/Sections/BlogPost.jsx]: what if we set blogData as the initial state like 
`const [blogs, setBlogs] = useState(blogData)` 
to remove this useEffect if we don't really need it?

File: package.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
     "eslint-config-next": "13.0.3",
     "next": "13.0.3",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "react-infinite-scroll-component": "^6.1.0"
```
Comment (Yvad60) [inline package.json]: it looks like we are not using package `"react-infinite-scroll-component"` anywhere will the infinite scrolling be handled a different way on mobile?

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Great work on the PR @sezeranoJchrisostome I noticed different features introduced in here. I left some comments let me know your decision about them

File: components/ui/Blog.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from "react"
+function Blog({date, title,description ,children}) {
+    return(
+        <div className=" flex-grow-0 min-w-48 md:min-w-94  md:max-w-94 min-h-83 md:p-5 p-4 lg:p-5 drop-shadow-custom-sm- border border-muted-400 rounded-xl flex flex-col">
```
Comment (serapieTuyishime) [inline components/ui/Blog.jsx]: the drop-shadow is not present, IMHO it is the `shadow-custom-sm-` causing it.

File: components/ui/Blog.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from "react"
```
Comment (serapieTuyishime) [inline components/ui/Blog.jsx]: this is not even used anywhere in the code.

Comment (serapieTuyishime) [inline components/ui/Blog.jsx]: this is not even used anywhere in the code.

Comment (serapieTuyishime) [inline components/ui/Blog.jsx]: This is not even used anywhere in the code.

Comment (serapieTuyishime) [inline components/ui/Blog.jsx]: This is not even used anywhere in the code.

File: tailwind.config.js [hunk 6]
```diff
@@ -1,74 +1,106 @@
...
+        "94":"29.8125rem",
+        "96":"47.31rem",
+      },
+      width: {
+        "70w": "70vw",
+        73: "18.125rem",
+        50: "50.375rem",
+        74: "19.375rem",
+      },
+      letterSpacing: {
```
Comment (serapieTuyishime) [inline tailwind.config.js]: We have two letterSpacing objects

File: package.json (hunk 3 above)
Comment (chrisostome01) [reply package.json]: Yeah, sure let clean them up

File: components/Sections/BlogPost.jsx (hunk 2 above)
Comment (chrisostome01) [reply components/Sections/BlogPost.jsx]: That can possibly work but for this am expecting that they will be more data, i kept that way

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Great job on this

**Review by Yvad60 — APPROVED** [review]
Sorry I thought I approved
Now Approved!

**Review by serapieTuyishime — APPROVED** [review]
Well done Mate!

**Pull request merged**